### ISSUE TITLE
SPEC

### ISSUE THREAD

**Issue by jayjayQy42** [WhiteeRabbit/Triton_RAT#6]
it saves as spec howfix?

**Comment by WhiteeRabbit** [comment]
Can you please write your problem in more detail :) . You can also write me via telegram: t.me/shamiloneleo

**Comment by jayjayQy42** [comment]
> Can you please write your problem in more detail :) . You can also write me via telegram: t.me/shamiloneleo

yeah sure so when i put telehram API bot token in and save and build it with the Command it saves as a SPEC file and not as an exe but when i compile to exe it doesnt work pls help thanks👍

**Comment by WhiteeRabbit** [comment]
Can you send the pyinstaller command (which you wrote on cmd to create the exe file). I tested with the (pyinstaller command which i wrote on the project repository https://github.com/WhiteeRabbit/Triton_RAT (in starting)). I tested this command several times in 2 windows virtual machines and it worked for me . This is the command i used: `pyinstaller --onefile --noconsole --uac-admin --hidden-import telebot --hidden-import pyautogui --hidden-import cv2 --hidden-import json --hidden-import base64 --hidden-import sqlite3 --hidden-import win32crypt --hidden-import Cryptodome.Cipher.AES --hidden-import shutil --hidden-import datetime --hidden-import numpy --hidden-import pyaudio --hidden-import wave --hidden-import random --hidden-import browser_cookie3 --hidden-import pyttsx3 --hidden-import pynput ` . After writing this command, the exe file should appear in the /dist folder. Try running cmd as administrator and then try this command if that didn't help. Like here⬇️
![1 (1)](https://github.com/user-attachments/assets/1c73ea92-381e-40db-9a09-4111f35ba230)
![1 (2)](https://github.com/user-attachments/assets/bd88d02b-7103-49c0-8327-862ff4f94696)

**Comment by jayjayQy42** [comment]
hi i did all the steps the build and dist create and the spec file is still there and the exe file to with the pythhon logo and when i open i go to telegram /start and it says password i did (acd) but it didnt work password is incorect 
so wgat now

**Comment by WhiteeRabbit** [comment]
Hello again, I am very glad that everything worked for you! And i apologize for the problems you encountered. Thank you very much for writing the problem about the password, I fixed it. If it's not difficult for you, can you please put a star on the project. Once again, I apologize for the problems you experienced. Thank you very much!
